reference/chromecast/ADBMobile.media.html

```diff
@@ -1582,7 +1582,7 @@ <h4 class="name" id=".setDelegate"><span class="type-signature">(static) </span>
     <p>Sets an instance of MediaDelegate to be used by the ADBMobile library.<br>
 The delegate object passed should have 2 functions.</p>
 <ol>
-<li><code>getCurrentPlaybackTime()</code> - Returns current playhead in seconds. For video-on-demand (VOD), the value is specified in seconds from the beginning of the media item. For live streaming, the value is specified as the number of seconds since midnight UTC on that day.</li>
+<li><code>getCurrentPlaybackTime()</code> - Returns current playhead in seconds. For video-on-demand (VOD), the value is specified in seconds from the beginning of the media item.  For live streaming, if the player does not provide information about the content duration, the value can be specified as the number of seconds since midnight UTC of that day. Note: When using progress markers, the content duration is required and the playhead needs to be updated as number of seconds from the beginning of the media item, starting with 0.</li>

```

Comment:
I think we can drop the note for this getter and keep it only in the context of updatePlayhead.

Reply:
Updated.